other/readme.html

```diff
@@ -136,7 +137,7 @@ <h2 id="recommendations">Recommendations for best performance</h2>
 					<li>the <a href="http://aspell.net/" target="_blank" rel="noopener">GNU Aspell</a> 0.50 or higher and <a href="https://ftp.gnu.org/gnu/aspell/dict/" target="_blank" rel="noopener">its dictionaries</a> for spell checking functionality.</li>
```

Comment:
Do we even support aspell? I don't remember.

Reply:
You need both aspell and pspell for spellcheck to work as far as I know.

Reply:
In 2.1 we also support [enchant](https://secure.php.net/enchant) for spell check.